### PR TITLE
minikube: upgrade to v0.28.2

### PR DESCRIPTION
**Pull request by tomerlange**
To test just ensure k8s test passes.

### CODE REVIEW

**Comment by nplanel** [comment]
@tomerlange could you revert this patch in your PR 4e8848ae804574fffc717fb1e86b6d40280a5fdd

**Comment by hunchback** [comment]
@nplanel - this change is orthogonal to PR downgrading iptables, the reason we changed to the new version of minikube was to address a race condition (@tomerlange - kindly add references to that issue). In any case as iptables 1.6.1 works well with the new minikube 0.28.2 and as iptable 1.6.1 is a standard part of fc27 I see no reason to change.

In future in the context of switching to fc28 (which officially only uses iptables 1.6.2) we would anyway upgrade to iptables 1.6.2 and then in that context could check the current minikube/kubernetes compatibility to iptables 1.6.2.

**Comment by nplanel** [comment]
@hunchback iptables-1.6.1 is workaround .... so if minikube has been updated, we switch back to the latest distribution package

**Comment by hunchback** [comment]
@tomerlange - kindly check if the new minikube works with iptables-1.6.2 - if so then add a 2nd commit which reverses https://github.com/skydive-project/skydive/commit/4e8848ae804574fffc717fb1e86b6d40280a5fdd

**Comment by tomerlange** [comment]
DONE

**Comment by lebauce** [comment]
@tomerlange @hunchback This PR broke the CI because it switched to localkube, so it was reverted. Please accept the invitation to the Github organization so that the tests are triggered for your PRs, then submit a new PR with the suggested changes. Thanks !

File: scripts/ci/install-minikube.sh [hunk 1]
```diff
@@ -101,8 +101,9 @@ stop() {
 start() {
         check_minikube
 
+	local args="--kubernetes-version $K8S_VERSION --bootstrapper=localkube"
```
Comment (lebauce) [inline scripts/ci/install-minikube.sh]: nit: please fix indentation.

In addition, this should be:
```
local args="--kubernetes-version $K8S_VERSION"
```

File: scripts/ci/install-minikube.sh [hunk 2]
```diff
@@ -101,8 +101,9 @@ stop() {
...
         if [ "$MINIKUBE_DRIVER" == "none" ]; then
-                local args="--vm-driver=none"
+                args="$args --vm-driver=none"
```
Comment (lebauce) [inline scripts/ci/install-minikube.sh]: Here you can do
```
args="$args --vm-driver=none --bootstrapper=localkube"
```

**Pull request merged**